controllers/noteController.js

```diff
@@ -0,0 +1,105 @@
+const Note = require('../models/noteModel');
+const User = require('../models/userModel');
+
+const viewNoteAccount = (req, res) => {
+
+    User
+        .findById(req.params.id)
+        .populate('notes')
+        .then(user => {
+            console.log(user.notes)
+            if (!user) {
+                res.status(404).json({ errMsg: "User Not Found" });
+            } else {
+                res.status(200).json(user);
+            }
+
+        })
+};
+
+
+const createNote = (req, res) => {
+    const { title, content } = req.body;
+    const { id } = req.params
+    const noteEntry = new Note({ ...req.body, user_id: id });
+
+    if( !title && !content ) {
```

Comment:
I would probably use || (or) here because with this. You can send a note through if it doesn't have a title OR if it doesn't have any content. 

This just checks if they are both empty